### DISCUSSION TITLE
Empty exif Title is not used when reindexing photos.

### DISCUSSION THREAD

**Discussion by stenrock** [photoprism/photoprism#4461] [Q&A]
I have a lot of photos where the title is empty and Photoprism invent titles from available data. This can be very wrong. So I added title to the photos in Lightroom and exported new ones that replaced the existing in Photoprism. Then I made a Complete Rescan. Still no empty titles did change, only titles with text. What is needed to achieve this?

Running Photoprism docker-compose in Debian 12.

Edited question title and text to include word "empty" because that is the problem. Sorry for any misunderstanding.

**Comment by graciousgrey** [comment]
Thank you for your feedback!

As far as I remember it is currently not possible to have an empty title. As PhotoPrism will automatically generate a title if it is empty. In the future we might be able to add more config options for title generation however we have other tasks with a higher priority to work on first.

**Reply by lastzero** [reply]
Note that if you wanted to behave the software in such a way, you'd need to set a title in every supported metadata standard /  format as otherwise some title field might be empty resulting in no title although some title in some format is set. Essentially you would be forced to set / sync the title in every format at the same time.

**Reply by lastzero** [reply]
If you don't want a title to be displayed in the search, you can use the mosaic view.
